index.js

```diff
@@ -9,31 +9,15 @@ function createError (code, message, statusCode = 500, Base = Error) {
   code = code.toUpperCase()
   !statusCode && (statusCode = undefined)
 
-  function FastifyError (a, b, c) {
+  function FastifyError (...args) {
     if (!new.target) {
-      return new FastifyError(...arguments)
+      return new FastifyError(...args)
     }
     this.code = code
     this.name = 'FastifyError'
     this.statusCode = statusCode
 
-    // more performant than spread (...) operator
-    switch (arguments.length) {
-      case 3:
-        this.message = format(message, a, b, c)
-        break
-      case 2:
-        this.message = format(message, a, b)
-        break
-      case 1:
-        this.message = format(message, a)
-        break
-      case 0:
-        this.message = message
```

Comment:
This case could be worth to be supported adding `if(args.length`)

Reply:
this is coverd by benchmark already, not worth it.

```ts
  .add('instantiate FastifyError', function () { new FastifyError() }, { minSamples: 100 }) // eslint-disable-line no-new
```